app/templates/tower_settings.html

```diff
@@ -104,7 +104,46 @@ <h1 style="display:inline">Tower settings:
 
 </div>
 
-<hr>
+
+<hr />
+
+{{ form.wheatley_enabled(class="d-none",id="wheatley_toggle") }}
+
+<h3>Wheatley</h3>
+<div class="form-row mt-4">
+    <div class="col-12 col-md-6 pr-2">
+        <div class="form-row my-2">
+            <div class="col-6 pt-1">
+                <h4>Enable Wheatley:</h4>
+            </div>
```

Comment:
I think some text about to describe what Wheatley is would be useful. Host mode has a nice description along side it below.

Reply:
Yep it's on my to-do list

Reply:
Done.